kratos/includes/model_part.h

```diff
@@ -1716,30 +1716,55 @@ class KRATOS_API(KRATOS_CORE) ModelPart final
     ///@}
     ///@name Access
     ///@{
+    
 
+    /**
+     * @brief Get reference to the ProcessInfo of root model part.
+     * @details calls pGetProcessInfo and dereference it
+     * @return reference to the ProcessInfo object.
+     */
     ProcessInfo& GetProcessInfo()
     {
-        return *mpProcessInfo;
+        return *this->pGetProcessInfo();
     }
 
+    /**
+     * @brief Get const reference to the ProcessInfo of root model part.
+     * @details Calls pGetProcessInfo and dereference it
+     * @return Const reference to the ProcessInfo object.
+     */
     ProcessInfo const& GetProcessInfo() const
     {
-        return *mpProcessInfo;
+        return *this->pGetProcessInfo();
     }
 
+    /**
+     * @brief Get pointer to the ProcessInfo of root model part.
+     * @return Pointer to the ProcessInfo object.
```

Comment:
No need to encode type info in the annotations. Doxygen already has the type information.

```suggestion
     * @brief Access the @ref ProcessInfo related to the current @ref ModelPart tree.
```